src/Map.jsx

```diff
@@ -34,16 +34,13 @@ class Map extends Component {
         minPitch: 0,
         maxPitch: 85,
       },
-      coordinates: [0, 0],
-      info: '',
+      coordinates: { lon: 0, lat: 0 },
```

Comment:
let's be more specific, what are these the coordinates of? We could put this in a `popup` object (so we'd do `popup.coordinates.lon`, or just rename it to popupCoordinates?